packages/core/src/sass/app/_common/components/notification-banner.scss

```
@@ -76,4 +76,13 @@
         top: 16px;
         cursor: pointer;
     }
+
+    @include mobile {
+        max-width: 328px;
```

Same here, you can do:
```suggestion
        max-width: calc(100% - 0.8rem);
        margin-right: 0.4rem;
```
So we wouldn't need the additional screen media query for `320px`.